regparser/commands/sxs_layers.py

```diff
@@ -11,7 +11,10 @@
 def previous_sxs(cfr_title, cfr_part, stop_version):
     """The SxS layer relies on all notices that came before a particular
     version"""
-    for previous_version in entry.FinalVersion(cfr_title, cfr_part):
+    version_ids = [
```

Comment:
This might be easier to read:
```sub_entries = entry.FinalVersion(cfr_title, cfr_part).sub_entries()
version_ids = [fv.path[-1] for fv in sub_entries]```